frontends/main/src/app-pages/DashboardPage/OrganizationContent.test.tsx

```diff
@@ -754,9 +752,25 @@ describe("OrganizationContent", () => {
           )?.id,
           course: { id: courses[0].id, title: courses[0].title },
         },
-        grades: [], // No grades = enrolled but not completed
```

Comment:
Notes:
- By default, enrollment factory creates a certificate. (We might want to revisit that choice.)
- Subsequent to #2792, the comment `grades: [], // No grades = enrolled but not completed` is innacurate. The enrollment in this test SHOULD be "Completed" since it has a cert but no grades.